index.md

```diff
@@ -16,6 +16,24 @@ This session aims to introduce the use of Docker containers with the goal of usi
 > If you are looking for a lesson on using *singularity* containers (instead of Docker), see this lesson:
 > * [Reproducible Computational Environments Using Containers: Introduction to Singularity](https://carpentries-incubator.github.io/singularity-introduction/)
 
+> ## A note about Docker
+>
+> Docker is a mature, robust and very widely used application. Nonetheless,
+> it is still under extensive development. New versions are released regularly
+> often containing a range of updates and new features.
+>
+> While we do our best to ensure that this lesson remains up to date and the
+> descriptions and outputs shown match what you will see on your own computer,
+> inconsistencies can occur.
+> 
+> If you spot inconsistencies or encounter any problems, please do report them
+> by [opening an issue] in the [GitHub repository] for this lesson.
+> 
+> [opening an issue]: https://github.com/carpentries-incubator/docker-introduction/issues/new
+> [GitHub repository]: https://github.com/carpentries-incubator/docker-introduction
```

Comment:
Since these links are reused, maybe instead of putting them at the bottom they should be added to `links.md`?